code/modules/projectiles/ammo_datums.dm

```diff
@@ -286,6 +286,12 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	shrapnel_chance = 25
 	sundering = 2.15
 
+/datum/ammo/bullet/pistol/heavy/ap
+	name = "armor piercing heavy pistol bullet"
+	hud_state = "pistol_ap"
+	damage = 40
```

Comment:
```suggestion
	damage = 40
```
AP should not deal higher damage, right? But I think it's for balance since, FC's got a special pistol on their hands...